### PR TITLE
Issue #15218: formatted input files for section 4.5.1 Where to break to follow google java style guide rules

### PR DESCRIPTION
**Pull request by Zopsss**
#15218

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputMethodParamPad.java [hunk 1]
```diff
@@ -3,224 +3,139 @@
...
+class InputMethodParamPad {
+  class Inner {
+    void testGenerics1
+      () // violation ''(' should be on the previous line.'
         {
-            Comparable
-                <
-                String
-                >
-                c = new String();
```
Comment (romani) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputMethodParamPad.java:16]: please restore all weird formatting, except for "{" that is not part of modules focus.
we need weird code only for https://checkstyle.org/google_style.html#a4.5.1 modules.

in next PRs you will create InputCorrentMethodParamPad and there will will use auto format to make sure we are not conflicting with formatter.

**Review by romani — CHANGES_REQUESTED** [review]
items:

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputMethodParamPad.java (hunk 1 above)
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputMethodParamPad.java:16]: done

**Review by romani — APPROVED** [review]
ok to merge

**Pull request merged**